javascript/modules/client/src/internal/queries/withdraw.ts

```diff
@@ -0,0 +1,24 @@
+import { gql } from "graphql-request";
+
+export const searchWithdraw = gql`
```

Comment:
I think we need some higher level renamings here...  It should be `withdrawals` or `getWithdrawals`... filters/search is assumed

Reply:
Changed, `searchWithdraw` to `QueryGetWithdrawals` and `searchDeposit` to `QueryGetDeposits`

Let me know if you are ok with the renaming @brickpop

Reply:
I'd personally use `QueryWithdrawals` and `QueryDeposits`, but if the underlying graphql query is named as `GetXXX` then, it's fine :+1: